src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/movements/TemporaryAbsenceRepositoryIntTest.kt

```diff
@@ -584,5 +609,35 @@ class TemporaryAbsenceIntTest(
         assertThat(scheduledTemporaryAbsence?.eventId).isNull()
       }
     }
+
+    @Test
+    fun `should handle missing direction for TAP external movements (bad data exists in NOMIS)`() {
```

Comment:
After reviewing data in NOMIS decided to add a test that shows we can handle temporary absences missing a direction code as there are some in prod data.